core/src/main/scala/io/projectglow/sql/SqlExtensionProvider.scala

```diff
@@ -146,5 +149,16 @@ object SqlExtensionProvider {
       FunctionIdentifier("vector_to_array"),
       exprs => VectorToArray(exprs.head)
     )
+
+    functionRegistry.registerFunction(
+      FunctionIdentifier("aggregate_by_index"),
```

Comment:
Tiny nit: can we restore the function ordering in this file? I think it's easiest to visually parse whether a function exists that way.

Reply:
Bump - I think it makes sense to keep `SqlExtensionProvider` sorted.